allennlp/common/checks.py

```diff
@@ -33,3 +36,16 @@ def check_dimensions_match(dimension_1: int,
     if dimension_1 != dimension_2:
         raise ConfigurationError(f"{dim_1_name} must match {dim_2_name}, but got {dimension_1} "
                                  f"and {dimension_2} instead")
+
+
+def cuda_is_valid(device_id: int):
```

Comment:
I'm not a fan of this name, to me `cuda_is_valid` sounds like a function that returns a boolean. I would prefer it were called `check_for_gpu` or something that's more descriptive when you call it.